docs/sources/flow/reference/components/otelcol.processor.filter.md

```diff
@@ -0,0 +1,260 @@
+    traces  = [otelcol.exporter.otlp.default.input]
+  }
+}
+```
+
+
+Some values in the River string are [escaped][river-strings]:
+* `\` is escaped with `\\`
+* `"` is escaped with `\"`
+
+### Drop non-HTTP spans and sensitive logs
+
+```river
+otelcol.processor.filter "default" {
+  error_mode = "ignore"
+
+  traces {
+    span = [
+      "attributes[\"http.request.method\"] == nil",
+    ]
+  }
+
+  logs {
+    log_record = [
+      "IsMatch(body, \".*password.*\")",
+      "severity_number < SEVERITY_NUMBER_WARN",
+    ]
+  }
+
+  output {
+    metrics = [otelcol.exporter.otlp.default.input]
+    logs    = [otelcol.exporter.otlp.default.input]
+    traces  = [otelcol.exporter.otlp.default.input]
+  }
+}
+```
+
+Each `"` is [escaped][river-strings] with `\"` inside the River string.
+
+
+Some values in the River strings are [escaped][river-strings]:
+* `\` is escaped with `\\`
+* `"` is escaped with `\"`
+
+[river-strings]: {{< relref "../../config-language/expressions/types_and_values.md/#strings" >}}
+
+
+[OTTL]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.85.0/pkg/ottl/README.md
+[OTTL Converter functions]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl/ottlfuncs#converters
+[HasAttrKeyOnDataPoint]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/filterprocessor/README.md#hasattrkeyondatapoint
+[HasAttrOnDataPoint]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/filterprocessor/README.md#hasattrondatapoint
+[OTTL booleans]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.85.0/pkg/ottl#booleans
+[OTTL math expressions]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.85.0/pkg/ottl#math-expressions
+[OTTL boolean expressions]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.85.0/pkg/ottl#boolean-expressions
+[OTTL resource context]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.85.0/pkg/ottl/contexts/ottlresource/README.md
+[OTTL scope context]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.85.0/pkg/ottl/contexts/ottlscope/README.md
+[OTTL span context]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.85.0/pkg/ottl/contexts/ottlspan/README.md
+[OTTL spanevent context]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.85.0/pkg/ottl/contexts/ottlspanevent/README.md
+[OTTL metric context]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.85.0/pkg/ottl/contexts/ottlmetric/README.md
+[OTTL datapoint context]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.85.0/pkg/ottl/contexts/ottldatapoint/README.md
+[OTTL log context]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.85.0/pkg/ottl/contexts/ottllog/README.md
```

Comment:
Are all of those URLs used in this doc page?

Reply:
Thanks for the review! I've removed the unused Markdown refs in latest commit.